include/cuda/std/detail/libcxx/include/version

```diff
@@ -201,234 +201,468 @@ __cpp_lib_void_t                                        201411L <type_traits>
 #endif
 
+
+// We unconditionally define `__cuda_lib_meow` so that there is only one place to set the value
 #if _LIBCUDACXX_STD_VER > 11
-# define __cpp_lib_chrono_udls                          201304L
-# define __cpp_lib_complex_udls                         201309L
+# define __cuda_lib_chrono_udls                          201304L
+# define __cuda_lib_complex_udls                         201309L
 #ifdef _LIBCUDACXX_IS_CONSTANT_EVALUATED
-# define __cpp_lib_constexpr_complex                    201711L
+# define __cuda_lib_constexpr_complex                    201711L
 #endif
-# define __cpp_lib_concepts                             202002L
-# define __cpp_lib_exchange_function                    201304L
-# define __cpp_lib_generic_associative_lookup           201304L
-# define __cpp_lib_integer_sequence                     201304L
-# define __cpp_lib_integral_constant_callable           201304L
-# define __cpp_lib_is_final                             201402L
-# define __cpp_lib_is_null_pointer                      201309L
-# define __cpp_lib_make_reverse_iterator                201402L
-# define __cpp_lib_make_unique                          201304L
-# define __cpp_lib_null_iterators                       201304L
-# define __cpp_lib_quoted_string_io                     201304L
-# define __cpp_lib_result_of_sfinae                     201210L
-# define __cpp_lib_robust_nonmodifying_seq_ops          201304L
+# define __cuda_lib_concepts                             202002L
+# define __cuda_lib_exchange_function                    201304L
+# define __cuda_lib_generic_associative_lookup           201304L
+# define __cuda_lib_integer_sequence                     201304L
+# define __cuda_lib_integral_constant_callable           201304L
+# define __cuda_lib_is_final                             201402L
+# define __cuda_lib_is_null_pointer                      201309L
+# define __cuda_lib_make_reverse_iterator                201402L
+# define __cuda_lib_make_unique                          201304L
+# define __cuda_lib_null_iterators                       201304L
+# define __cuda_lib_quoted_string_io                     201304L
+# define __cuda_lib_result_of_sfinae                     201210L
+# define __cuda_lib_robust_nonmodifying_seq_ops          201304L
 # if !defined(_LIBCUDACXX_HAS_NO_THREADS)
-#   define __cpp_lib_shared_timed_mutex                 201402L
+#   define __cuda_lib_shared_timed_mutex                 201402L
 # endif
-# define __cpp_lib_span                                 201902L
-# define __cpp_lib_string_udls                          201304L
-# define __cpp_lib_transformation_trait_aliases         201304L
-# define __cpp_lib_transparent_operators                201210L
-# define __cpp_lib_tuple_element_t                      201402L
-# define __cpp_lib_tuples_by_type                       201304L
-#endif
+# define __cuda_lib_span                                 201902L
+# define __cuda_lib_string_udls                          201304L
```

Comment:
We don't have string (yet), so should sit down and do a pass over these and remove things we don't support in `cuda_std` mode?

Reply:
I can see how this made things more difficult when defining the `cpp_lib` macros as there wouldn't be a value or macro set for them. I'm not a fan of special cases.

Reply:
I took this up and separated them in case we do not implement something.

I believe this is not too bad, as we really want to track what we implement and what not